Content.Server/Shuttles/Systems/ArrivalsSystem.cs

```diff
@@ -299,6 +380,11 @@ public override void Update(float frameTime)
 
                     if (targetGrid != null)
                         _shuttles.FTLTravel(uid, shuttle, targetGrid.Value, dock: true);
+
+                    // The ArrivalsCooldown includes the trip there, so we only need to add the time taken for
+                    // the trip back.
+                    comp.NextArrivalsTime = _ticker.RoundDuration() +
+                         TimeSpan.FromSeconds(_cfgManager.GetCVar(CCVars.ArrivalsCooldown) + tripTime);
```

Comment:
Shouldn't be using RoundDuration and also += like NextTransfer is.

Reply:
I removed roundDuration. However it doesn't make sense to use +=  here because the time estimate is not based on the old time estimate.